include/blockwrapper.hpp

```diff
@@ -93,6 +94,17 @@ template <class T> struct BlockWrapper {
           static_cast<CBHelpProc>([](CBlock *b) { return CBOptionalString(); });
     }
 
+    // properties
+    if constexpr (has_properties<T>::value) {
+      result->properties =
+          static_cast<CBPropertiesProc>([](CBlock *b) -> const CBTable * {
+            return reinterpret_cast<BlockWrapper<T> *>(b)->block.properties();
+          });
+    } else {
+      // properties is optional!
+      result->properties = nullptr;
```

Comment:
Since Rust is implementing it with none and returning a null table
We could just do the same here and return a null table.
Usually in the case above, rust side I implement a `hasCompose` etc for example.

Reply:
Correct, let's go with the null table.